### ISSUE TITLE
1.5.x feature request: Make var-value declarations "organizable"

### ISSUE THREAD

**Issue by strgaltdel** [FrSkyRC/ETHOS-Feedback-Community#3133]
the new GVAR implementation allows "tons" of combinations and offers huge freedome.

So it's up to the user to plan the order of declarations in the right order:
As long as declarations do not  follow an exact "only one can be active at the same time" path, 
the active value is choosen by a "first serve, first win" strategy

actually this leads into a situation that in case you want to change the priorizing users will have to redefine (most) of the given declarations.
So it would be more convenient to offer the typical "move / copy / paste / clone" pop up as a tool to organize the declarations.

Not the best example, but imagine  user wants change order and priorize FM "start", which can be active at the same time when SA is down:

![moveVAR](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/20392683/9de00bda-6dbd-41e7-be64-f474ecc1bff4)

imho not a "must have" for the first 1.5 release, but maybe later on

thanks & regards

**Comment by strgaltdel** [comment]
..keep alive comment

**Comment by Toffs** [comment]
Yes, agreed this would be nice quality of life improvement one day :)

**Comment by strgaltdel** [comment]
.. keep alive

**Comment by strgaltdel** [comment]
"ping"

**Comment by strgaltdel** [comment]
"ping"
